static/css/inline-selection.scss

```diff
@@ -2,108 +2,33 @@
 @import "partials/mixins";
 @import "partials/buttons";
 @import "partials/theme";
-
+@import "partials/animations";
 // This is how far outside the selection the mover target extends:
-$mover-outer: 10px;
+$mover-outer: 30px;
 
 // And this is how far inside the selection it extends:
-$mover-inner: 40px;
+$mover-inner: 30px;
 
 // This is the size of the little blue handles:
-$handle-size: 25px;
+$handle-size: 16px;
 
 // And the size when the selection is "small":
 $handle-size-small: 10px;
 
-// This is how far out of the box the handles go:
-$handle-outside-extent: 0;
-
 // And some calculated values based on these that will be used below:
-$handle-mover-offset: $mover-outer - $handle-outside-extent / 2;
+$handle-mover-offset: -1px;
 $mover-size: $mover-outer + $mover-inner;
 $overlay-z-index: 9999999999;
 
-.highlight {
-  position: absolute;
-  cursor: move;
-  border: 1px dashed rgba($link-blue, 0.5);
-  border-radius: 2px;
-  box-shadow: 0 0 0 1px rgba(255, 255, 255, 0.4);
-  z-index: $overlay-z-index;
-}
-
-.highlight-buttons {
-  @include flex-container(row, center, center, wrap);
-  position: absolute;
-  bottom: -55px;
-  right: 0;
-  z-index: 6;
-
-  .bottom-selection & {
-    bottom: 5px;
-  }
-}
+$modal-color: rgba(0, 0, 0, 0.7);
+$very-light-grey: #ededed;
 
-.highlight-button-cancel {
-  @extend .button;
-  @extend .secondary;
-  // Overrides display: flex from .button, which causes hover problems
-  display: block;
-  background: $very-light-grey url("MOZ_EXTENSION/icons/cancel.svg") center center no-repeat;
-  background-size: 18px 18px;
-  margin: 5px;
-  width: 40px;
-
-  &:hover,
-  &:focus,
-  &:active {
-    background: darken($very-light-grey, 7) url("MOZ_EXTENSION/icons/cancel.svg") center center no-repeat;
-    background-size: 18px 18px;
-  }
-}
-
-.highlight-button-save {
-  @extend .button;
-  @extend .primary;
-  font-size: 18px;
-  margin: 5px;
-  min-width: 80px;
-}
-
-.highlight-button-download {
-  @extend .button;
-  @extend .secondary;
-  // Overrides display: flex from .button, which causes hover problems
-  display: block;
-  background: $very-light-grey url("MOZ_EXTENSION/icons/download.svg") center center no-repeat;
-  background-size: 18px 18px;
-  margin: 5px;
-  width: 40px;
-
-  &:hover,
-  &:focus,
-  &:active {
-    background: darken($very-light-grey, 7) url("MOZ_EXTENSION/icons/download.svg") center center no-repeat;
-    background-size: 18px 18px;
-  }
-}
-
-.highlight-button-cancel,
-.highlight-button-download {
-  opacity: 0.9;
-  transition: opacity 250ms;
-
-  &:hover,
-  &:focus,
-  &:active {
-    opacity: 1;
-  }
-}
 
 .mover-target {
+  @include flex-container(row, center, center);
```

Comment:
Note some of the flex container things have been causing problems with interactive elements.  I've had to remove it from several buttons.  Is there a reason why this is flex?